thetis/options.py

```diff
@@ -372,9 +372,9 @@ class CommonModelOptions(FrozenConfigurable):
     use_lax_friedrichs_velocity = Bool(
         True, help="use Lax Friedrichs stabilisation in horizontal momentum advection.").tag(config=True)
     lax_friedrichs_velocity_scaling_factor = FiredrakeConstantTraitlet(
-        Constant(1.0), help="Scaling factor for Lax Friedrichs stabilisation term in horiozonal momentum advection.").tag(config=True)
+        Constant(1.0), help="Scaling factor for Lax Friedrichs stabilisation term in horizontal momentum advection.").tag(config=True)
     use_lax_friedrichs_tracer = Bool(
-        True, help="Use Lax Friedrichs stabilisation in tracer advection.").tag(config=True)
+        False, help="Use Lax Friedrichs stabilisation in tracer advection.").tag(config=True)
```

Comment:
My concern here is that this changes the default behavior for the 2D model as well. For 3D model, omitting the LF flux for tracers seems like the right thing to do, I'm not sure about 2D. If needed, we could specify this option separately for 2D and 3D. Any preferences?

Reply:
I think for the (limited number of) 2d cases with tracers that we've run so far, we've had LF switched off - as it's fairly diffusive. So I'm happy for this to be the default